[build] couldn't load source package: /build/source/nil/client/client.go:209:28: intTxn.MarshalSSZ undefined

```
$ nix develop
error: builder for '/nix/store/l85y7azl3hy6czl2iw9dzvsdpp0sip06-nil-go-modules.drv' failed with exit code 2;
       last 25 log lines:
       > make: *** [nil/Makefile.inc:18: nil/internal/vm/state_generated_mock.go] Error 1
       > make: *** Waiting for unfinished jobs....
       > couldn't load source package: /build/source/nil/client/client.go:209:28: intTxn.MarshalSSZ undefined (type *"github.com/NilFoundation/nil/nil/internal/types".InternalTransactionPayload has no field or method MarshalSSZ) (and 1 more errors)
       > moq [flags] source-dir interface [interface2 [interface3 [...]]]
       >   -fmt string
       >          go pretty-printer: gofmt, goimports or noop (default gofmt)
       >   -out string
       >             output file (default stdout)
       >   -pkg string
       >            package name (default will infer)
       >   -rm
       >       first remove output file, if it exists
       >   -skip-ensure
       >         suppress mock implementation check, avoid import cycle if mocks generated outside of the tested package
       >   -stub
       >       return zero values when no mock implementation is provided, do not panic
       >   -version
       >           show the version for moq
       >   -with-resets
       >       generate functions to facilitate resetting calls made to a mock
       > Specifying an alias for the mock is also supported with the format 'interface:alias'
       > Ex: moq -pkg different . MyInterface:MyMock
       > exit status 1
       > client.go:17: running "go": exit status 1
       > make: *** [nil/Makefile.inc:15: nil/client/client_generated_mock.go] Error 1
       For full logs, run 'nix log /nix/store/l85y7azl3hy6czl2iw9dzvsdpp0sip06-nil-go-modules.drv'.
error: 1 dependencies of derivation '/nix/store/86xxycrrq7s0a5mj5magcywsva87clww-nil-env.drv' failed to build
```

--- a/nil/Makefile.inc
+++ b/nil/Makefile.inc
@@ -11,10 +11,10 @@ generate_mocks: \
 	$(root_db)/rwtx_generated_mock.go \
 	$(root_db)/db_generated_mock.go
 
-$(root_client)/client_generated_mock.go: $(root_client)/client.go
+$(root_client)/client_generated_mock.go: $(root_client)/client.go ssz_types
 	cd $(root_client) && go generate
 
-$(root_vm)/state_generated_mock.go: $(root_vm)/interface.go
+$(root_vm)/state_generated_mock.go: $(root_vm)/interface.go ssz_types
 	cd $(root_vm) && go generate
 
 $(root_db)/rwtx_generated_mock.go: $(root_db)/kv.go ssz_types

